cmd/queue/main.go

```diff
@@ -198,6 +205,11 @@ func probeUserContainer() bool {
 // Make handler a closure for testing.
 func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, handler http.Handler) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		recorder := pkghttp.NewResponseRecorder(w, http.StatusOK)
+		reqCtx, fullQueueProxySpan := trace.StartSpan(r.Context(), "queueproxy")
+		defer fullQueueProxySpan.End()
+		r.Header.Set(network.ProxyHeaderName, activator.Name)
```

Comment:
This will break autoscaling big time, because the code below will assume that all requests are proxied. Please remove it.

Reply:
thanks - removed.